gtruths/tracks/eufs/small_track/small_track.csv

```diff
@@ -0,0 +1,72 @@
+x,y,color
```

Comment:
Just to confirm, these track names are the same track names you choose on the simulators?

Reply:
Yes, easier to understand which one is which when using the sim I think